src/deskolem.ml

```diff
@@ -0,0 +1,49 @@
+open Core
+open Terms
+open Timed
+
+(** [subst_inv t s x] replaces the term starting with the symbol [s] by a fresh
```

Comment:
"the term" -> which term? do you mean every term?

Reply:
The term which has the shape: `s t1 t2 ... t3`.

Reply:
Should not it be "all subterms starting with the symbol [s]"?

Reply:
Changed in 64da854cb5e274e46628cedb8ed76826e6a7086.